core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/support/IncludeExclude.java

```diff
@@ -517,6 +644,11 @@ public boolean isRegexBased() {
         return include != null || exclude != null;
     }
 
+    public boolean isPartitionFilter() {
```

Comment:
nit: I wonder if we should name this `isPartitionBased()` to be consistent with the `isRegexBased()`?